components/table/table-wrapper.js

```diff
@@ -381,6 +391,25 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 		});
 	}
 
+	_checkSiblingSortableCells(c) {
+		const nodes = Array.from(c.childNodes);
+		const isSortableCell = nodes.some((element) => element.localName === 'd2l-table-col-sort-button');
```

Comment:
I'd recommend an early return instead, since there isn't anything that happens if it's not a sortable cell, e.g., 
```suggestion
		const isSortableCell = nodes.some((element) => element.localName === 'd2l-table-col-sort-button');
		if (!isSortableCell) return;
```